### PR TITLE
move cloudwatch log group module

### LINKED ISSUE #32: cloudwatch log group

### PR DESCRIPTION
**Pull request by ryanking**
Fixes #32

### CODE REVIEW

File: aws-cloudwatch-log-group/variables.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+variable "project" {
+  type        = "string"
+  description = "The high-level project. [Doc](https://czi.quip.com/TBxAAAo70sa8#HJSACAjH8mS)"
```
Comment (edulop91) [inline aws-cloudwatch-log-group/variables.tf]: these are private links

File: aws-params-writer/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Sets unencrypted string parameters in the AWS SSM parameter store. Often used in combination with Chamber
```
Comment (edulop91) [inline aws-params-writer/README.md]: should this be `encrypted`?

File: aws-cloudwatch-log-group/variables.tf (hunk 1 above)
Comment (ryanking) [reply aws-cloudwatch-log-group/variables.tf]: whoops, these were the old one

File: aws-params-writer/README.md (hunk 2 above)
Comment (ryanking) [reply aws-params-writer/README.md]: yeah

**Pull request merged**